setup.cfg

```
@@ -72,6 +72,9 @@ plugins =
 
 [coverage:report]
 precision = 2
+fail_under = 97.33
```

```suggestion
fail_under = 97
```

Just a small suggestion here - could we just round down to the nearest integer? I know our eventual goal is 100% (encode/uvicorn#102), but in the meantime, do we need the decimal places? I'm thinking about new contributors I guess. It's a little intimidating to see all the CI checks fail just because coverage decreased by 0.05%.

---

There was an issue on coverage. I didn't notice at the time. I've fixed it on https://github.com/nedbat/coveragepy/pull/1317. It only took me 9 months to realize the issue. 😎

---

The thing is that over the year, there were countless PRs dropping the percentage... It feels kind tiring have to fix those. 🙁

---

And one last thing... This will be useful as self motivation. I'll get this to 100% fast enough! 👨‍🏭🔨